bayesian_cnn_prometheus/analysis/masks_analyzer.py

```diff
@@ -0,0 +1,63 @@
+import json
+import os
+import numpy as np
+
+from bayesian_cnn_prometheus.analysis.similarity_comparer import SimilarityComparer
+from typing import List
+
+
+class MasksAnalyzer:
+    def __init__(self, model_name: str, lesion_masks_path: str, variance_masks_path: str):
+        """
+        Creates MasksAnalyzer object to compare generated variance masks with respective lesion masks.
+        :param model_name: name of the model that has generated variance masks
+        :param lesion_masks_path: path to a file with original lesion masks
+        :param variance_masks_path: path to a file with generated variance masks
```

Comment:
Wiem, że to moja wina, bo nie wysłałam Ci więcej masek, ale ... :D 
Tutaj w docstringu jest napisane "path to file",  a wg kodu powinien być path to dir.

Reply:
Katalog to też plit...
A poważnie to racja, poprawię